spec/minipack/manifest_spec.rb

```diff
@@ -16,11 +16,10 @@
   describe '#lookup_pack_with_chunks!' do
     subject { described_class.new(path, cache: false).lookup_pack_with_chunks!(name, type: type) }
 
-    let(:path) { File.expand_path('../support/files/manifest.json', __dir__) }
-    let(:type) { nil }
-
     context 'with name with ext' do
+      let(:path) { File.expand_path('../support/files/manifest.json', __dir__) }
       let(:name) { 'application.js' }
+      let(:type) { nil }
```

Comment:
Since this seems like a default, you can probably leave it up in the `describe` scope and override it as needed (like on line 37)